frontend/src/Editor/Components/Table/columns/index.jsx

```diff
@@ -164,7 +164,10 @@ export default function generateColumnsData({
                       }
                     }}
                     onBlur={(e) => {
-                      if (e.target.defaultValue !== e.target.value) {
+                      if (
+                        e.target.defaultValue !== e.target.value &&
+                        e.target.defaultValue.replace(/\n/g, '') !== e.target.value // do we require this, I have a doubt, need to add this condition, only when we change the column data type to string from text
```

Comment:
I checked for the string and default datatype. Seems we're getting the right value and this can be removed